java/code/src/com/redhat/rhn/common/util/FileUtils.java

```diff
@@ -145,10 +147,10 @@ public static String readStringFromFile(String path, boolean noLog) {
             return contents;
         }
         catch (FileNotFoundException e) {
-            throw new RuntimeException("File not found: " + path);
```

Comment:
We should never throw `RuntimeException` directly... looks like it has bitten us now.